src/components/elements/annotationSidePanel/components/annotationWizard/steps/AnnotationSelectInstanceStep.tsx

```diff
@@ -0,0 +1,158 @@
+/* Import Dependencies */
+import { isEmpty } from 'lodash';
+import jp, { PathComponent } from 'jsonpath';
+import { Row, Col } from 'react-bootstrap';
+
+/* Import Hooks */
+import { useAppSelector } from 'app/Hooks';
+
+/* Import Store */
+import { getAnnotationTarget } from 'redux-store/AnnotateSlice';
+
+/* Import Types */
+import { DigitalSpecimen } from 'app/types/DigitalSpecimen';
+import { DigitalMedia } from 'app/types/DigitalMedia';
+import { Dict } from 'app/Types';
+
+/* Import Components */
+import NewInstance from './NewInstance';
+import ExistingInstance from './ExistingInstance';
+
+
+/* Props Type */
+type Props = {
+    superClass: DigitalSpecimen | DigitalMedia | Dict,
+    schemaTitle?: string,
+    formValues?: Dict,
+    SetFieldValue?: Function
```

Comment:
Can this be made more specific?

Reply:
Its just the typing of the params, an explanation is provided in the function description.
formValues could be more specific, but its already an optional (but required) param due to how props are passed onto tab components. What would you like?

Reply:
Can you be more specific with the `Function` type? Not sure if that's possible

Reply:
In python you specify what the Dict is of, is that a thing in typescript? otherwise leave as is